packages/types/src/modules-sdk/index.ts

```diff
@@ -114,7 +114,40 @@ export type ModuleJoinerConfig = Omit<
   isLink?: boolean // If the module is a link module
   linkableKeys?: string[] // Keys that can be used to link to other modules
   isReadOnlyLink?: boolean // If true it expands a RemoteQuery property but doesn't have a pivot table
-  tableName?: string // Name of the pivot table. If not provided it is auto generated
+  databaseConfig?: {
+    tableName?: string // Name of the pivot table. If not provided it is auto generated
+    idPrefix?: string // Prefix for the id column. If not provided it is "link"
+    schema?: Record<
```

Comment:
suggestion(non-blocking): `fields` or `extraFields`?

Reply:
will change it to `extraFields`